src/controller/c/motor.c

```diff
@@ -533,6 +537,18 @@ double wb_motor_get_available_torque(WbDeviceTag tag) {
   return NAN;
 }
 
+double wb_motor_get_multiplier(WbDeviceTag tag) {
+  double multiplier = NAN;
+  robot_mutex_lock_step();
+  Motor *m = motor_get_struct(tag);
+  if (m)
+    multiplier = m->multiplier;
+  else
+    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
```

Comment:
This saves an unneeded double initialization:
```suggestion
  double multiplier;
  robot_mutex_lock_step();
  Motor *m = motor_get_struct(tag);
  if (m)
    multiplier = m->multiplier;
  else {
    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
    multiplier = NAN
  }
```